syntax = "proto3";

package account;

import "google/protobuf/timestamp.proto";

message AccountInfo {
	string name = 1;
	string email = 2;
	string cpf = 3;
	string car_plate = 4;
	bool is_driver = 5;
}

message Account{
	string account_id = 1;
	AccountInfo info = 2;
	google.protobuf.Timestamp created_at = 3;
}